src/rmedia.c

```diff
@@ -702,7 +704,16 @@ MediaContext* LoadMediaContext(const char* fileName, MediaStreamReader streamRea
 				ctx->videoOutputImage.mipmaps = 1;
 				ctx->videoOutputImage.format  = PIXELFORMAT_UNCOMPRESSED_R8G8B8;
 
-				ImageClearBackground(&ctx->videoOutputImage, BLANK),
+				ImageClearBackground(&ctx->videoOutputImage, BLANK);
+
+				int allocatedBytes = av_image_alloc(ctx->alignedRgbData, ctx->alignedRgbLinesize, codecCtx->width, codecCtx->height, AV_PIX_FMT_RGB24, 32);
```

Comment:
This allocates a single contiguous buffer, and the other pointers (which will always be `NULL` for RGB24) point into that buffer; [documentation ref](https://ffmpeg.org/doxygen/6.0/group__lavu__picture.html#ga841e0a89a642e24141af1918a2c10448))

![image](https://github.com/user-attachments/assets/2215f817-2d6b-4c32-ac9b-a83ad488b0e2)